python/mxnet/ndarray/contrib.py

```diff
@@ -605,3 +605,27 @@ def multi_mp_adamw_update(weights, grads, mean, var, weights32, rescale_grad, lr
                                                     etas=etas,
                                                     name=name,
                                                     **kwargs)
+
+def multi_lamb_update(weights, grads, mean, var, temp_g, step_count,
+                      out=None, name=None, num_tensors=0, **kwargs):
```

Comment:
Could you add some documentation on the input types for these two functions? Otherwise it shows nothing on the API website